### ISSUE TITLE
`start_present` and `stop_present` aren't called by Servo

### ISSUE THREAD

**Issue by asajeffrey** [servo/rust-webvr#56]
Not sure if this is a servo issue or a rust-webvr issue, but the methods to start and stop presentation on a device are never called by servo. Shouldn't they be?

**Comment by jdm** [comment]
They're called internally in each backend:
* https://github.com/servo/rust-webvr/blob/1280d8a8078fc9a0c13233c6c691d9a9d7e8aada/rust-webvr/src/api/vrexternal/android/display.rs#L166
* https://github.com/servo/rust-webvr/blob/7b292186d85af794fb011e0c77b67acfdc69902e/rust-webvr/src/api/googlevr/display.rs#L113
* https://github.com/servo/rust-webvr/blob/7b292186d85af794fb011e0c77b67acfdc69902e/rust-webvr/src/api/oculusvr/display.rs#L131

**Comment by asajeffrey** [comment]
Okay, that deals with start, what about stop? E.g. if there are system resources which need cleaned up?

**Comment by MortimerGoro** [comment]
yes, ideally we can make servo call those functions when WebXR sessions are started or stopped

**Comment by asajeffrey** [comment]
Created a Servo issue, since that's where the problem is... https://github.com/servo/servo/issues/22924
